Lib/unittest/mock.py

```diff
@@ -1005,6 +1005,11 @@ def _get_child_mock(self, /, **kw):
         if _new_name in self.__dict__['_spec_asyncs']:
             return AsyncMock(**kw)
 
+        if self._mock_sealed:
+            attribute = "." + kw["name"] if "name" in kw else "()"
```

Comment:
nit
```suggestion
            attribute = f".{kw['name']} if "name" in kw else "()"
```

Reply:
Not sure this is better, why do you feel the f-string makes things better here?